src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -129,6 +143,58 @@ private ModelTypeTable createTypeTable(String implicitPackageName) {
         new CSharpModelTypeNameConverter(implicitPackageName));
   }
 
+  private PackageInfoView generateCsProjView(GapicInterfaceContext context) {
+    Model model = context.getModel();
+    GapicProductConfig productConfig = context.getProductConfig();
+    PackageInfoView.Builder view = PackageInfoView.newBuilder();
+    view.templateFileName(CSPROJ_TEMPLATE_FILENAME);
+    String outputPath = pathMapper.getOutputPath(context.getInterface(), productConfig);
+    view.outputPath(outputPath + File.separator + productConfig.getPackageName() + ".csproj");
+    view.fileHeader(fileHeaderTransformer.generateFileHeader(context));
+    view.serviceTitle(model.getServiceConfig().getTitle());
+    view.serviceDescription(model.getServiceConfig().getDocumentation().getSummary().trim());
+    view.domainLayerLocation(productConfig.getDomainLayerLocation());
+    view.authScopes(new ArrayList<String>()); // Unused in C#
+    view.releaseLevel(productConfig.getReleaseLevel());
+    String versionSuffix;
+    switch (productConfig.getReleaseLevel()) {
+      case ALPHA:
+        versionSuffix = "-alpha01";
```

Comment:
Will there ever have a "-alpha02"? If so, I assume it would require manual change on top of the genereated csproj file?

Reply:
Yes, changing to -alpha02 will need a manual change to the generated csproj file. There is a TODO in the generated file about fixing the version.